src/features/notice/ui/NoticeForm/NoticeForm.tsx

```
@@ -0,0 +1,132 @@
+	useEffect(() => {
+		if (initialData) {
+			form.setFieldsValue({
+				title: initialData.title,
+				content: initialData.content,
+				priority: initialData.priority,
+				date: dayjs(initialData.date),
+			});
+		}
+	}, [initialData, form]);
+
+	const handleSubmit = async (values: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)